src/handlers/expenditures/expenditureAdded.ts

```diff
@@ -0,0 +1,35 @@
+import { mutate } from '~amplifyClient';
+import {
+  CreateExpenditureDocument,
+  CreateExpenditureMutation,
+  CreateExpenditureMutationVariables,
+  ExpenditureStatus,
+} from '~graphql';
+import { ContractEvent } from '~types';
+import { getExpenditureDatabaseId, toNumber, verbose } from '~utils';
+
+export default async (event: ContractEvent): Promise<void> => {
+  const { contractAddress: colonyAddress } = event;
+  const { agent: ownerAddress, expenditureId } = event.args;
+  const convertedExpenditureId = toNumber(expenditureId);
+
+  verbose(
+    'Expenditure with ID',
+    convertedExpenditureId,
+    'added in Colony:',
+    colonyAddress,
+  );
+
+  await mutate<CreateExpenditureMutation, CreateExpenditureMutationVariables>(
+    CreateExpenditureDocument,
+    {
+      input: {
+        id: getExpenditureDatabaseId(colonyAddress, convertedExpenditureId),
+        colonyId: colonyAddress,
```

Comment:
maybe we should checksum this address, just to make sure they're all in the same format.

We've had trouble with them before being checksummed/not being checksummed or being all lower case. Checksumming will allow us to have one consistent way of storing them.